src/api/java/com/minecolonies/api/configuration/CommonConfiguration.java

```diff
@@ -406,6 +407,7 @@ protected CommonConfiguration(final ForgeConfigSpec.Builder builder)
         disableCitizenVoices = defineBoolean(builder, "disablecitizenvoices", false);
         diseaseModifier = defineInteger(builder, "diseasemodifier", 5, 1, 100);
         generateSupplyLoot = defineBoolean(builder, "generatesupplyloot", true);
+        forceLoadColony = defineBoolean(builder, "forceloadcolony", false);
```

Comment:
rename to chunk load colony, easier understandable

Reply:
I wanted to avoid calling it that because people will confuse it with "my colony is always chunkloaded with it even I'm not in range or offline"

Reply:
ok